### PR TITLE
Prepare 3.0.0

### PR DESCRIPTION
**Pull request by ralphtheninja**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,25 @@
 
 ## [Unreleased]
 
+## [3.0.0] - 2018-05-23
+
+### Added
+* Add node 10 to Travis (@ralphtheninja)
+* Add `UPGRADING.md` (@ralphtheninja)
```
Comment (ralphtheninja) [inline CHANGELOG.md]: Should link to this from README. Will update.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,25 @@
...
+
+### Changed
+* Update `standard` to `^11.0.0` (@ralphtheninja)
+* Update `encoding-down` to `~5.0.0` (@ralphtheninja)
+* Update `leveldown` to `^4.0.0` (@ralphtheninja)
+* Update `levelup` to `^3.0.0` (@ralphtheninja)
+* Change from `License & Copyright` to `License` in README (@ralphtheninja)
```
Comment (vweevers) [inline CHANGELOG.md]: Can simplify "Change from" to "Change"

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,25 @@
...
+* Replace `const` with `var` for IE10 support (#54) (@ralphtheninja)
```
Comment (vweevers) [inline CHANGELOG.md]: Leftover `#54`

Comment (ralphtheninja) [reply CHANGELOG.md]: I actually added it, but will remove :)

**Pull request merged**